### PR TITLE
f5 integration

### PR DESCRIPTION
**Pull request by aviadl**
First commit for F5 integration - only authentication works, both A and basic

DO NOT MERGE

related to https://github.com/demisto/server/issues/4846

### CODE REVIEW

**Comment by Spivaka** [comment]
@aviadl please review this

File: Integrations/integration-F5.yml [hunk 1]
```diff
@@ -0,0 +1,191 @@
+commonfields:
+  id: F5
+  version: -1
+name: F5
+display: F5
```
Comment (liorkol) [inline Integrations/integration-F5.yml]: Add system: true

File: Integrations/integration-F5.yml [hunk 2]
```diff
@@ -0,0 +1,191 @@
+commonfields:
+  id: F5
```
Comment (liorkol) [inline Integrations/integration-F5.yml]: Name is missing the exact product name

File: Integrations/integration-F5.yml [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+description: Integrate with F5
```
Comment (liorkol) [inline Integrations/integration-F5.yml]: Needs a bit more ...

**Pull request merged**